app/api/resources/user.py

```diff
@@ -85,15 +84,15 @@ class OtherUser(Resource):
     @users_ns.expect(auth_header_parser)
     @users_ns.response(HTTPStatus.OK.value, "Success.", public_user_api_model)
     @users_ns.response(
-        HTTPStatus.UNAUTHORIZED.value,
-        "%s\n%s\n%s"
-        % (
-            messages.TOKEN_HAS_EXPIRED,
-            messages.TOKEN_IS_INVALID,
-            messages.AUTHORISATION_TOKEN_IS_MISSING,
+        HTTPStatus.UNAUTHORIZED,
```

Comment:
@Sanchit-Trivedi it should be `HTTPStatus.UNAUTHORIZED.value`. Same for other places too.